stl/inc/ranges

```diff
@@ -1399,6 +1399,10 @@ namespace ranges {
             }
         }
 
+        _NODISCARD_EMPTY_MEMBER constexpr bool empty() const noexcept(noexcept(_Value == _Bound)) /* strengthened */ {

```

Comment:
`_NODISCARD_EMPTY_MEMBER` mentions that `clear()` may be intended, which is not a thing for `iota_view`:

https://github.com/microsoft/STL/blob/46843b3bf320c6070ff63e978936bcf898e5a2bc/stl/inc/yvals_core.h#L596-L600

Other views use plain `_NODISCARD`:

https://github.com/microsoft/STL/blob/46843b3bf320c6070ff63e978936bcf898e5a2bc/stl/inc/ranges#L1833